payloads/start.go

```diff
@@ -127,6 +127,9 @@ type RequestedResource struct {
 	// Value specifies the integer value associated with that resource.
 	Value int `yaml:"value"`
 
+	// ValueString is an optional string format value instead of integer
+	ValueString string `yaml:"value_string,omitempty"`
```

Comment:
Yes it probably will.  I thought about this as I was doing this simple first string which is directly comparable.  But I want to wait for the specific instance of non-comparable data to have a proof point of what a suitable interface would need to cover.  I don't want to architect too much without that usage data, because the interface ends up being an API that could need versioned or paint us into a corner.  We want to be able to add optional implementations of resource requests but always keep existing workload definitions functional.  I don't want resource A using version 1 of the comparison interface and resource B using version 2 of the comparison interface.  Plus I feel like I could make an argument for always in the future choosing to articulate resource requests as a list of requests which are each key:singlevalue pairs and the values always being comparable, but I'm also expecting some use case to come along and prove me wrong.